ui/src/org.ts

```diff
@@ -378,7 +385,8 @@ export async function fetchMembers(
   wallet: wallet.Wallet,
   gnosisSafeAddress: string
 ): Promise<OrgMembers> {
-  const response: MemberResponse = await graph.getGnosisSafeMembers(
+  const response: MemberResponse = await Safe.getMetadata(
```

Comment:
We can omit the explicit type annotation.

Reply:
6f609f6943e714b73f452722231eff6caf14c608